velox/dwio/common/CMakeLists.txt

```diff
@@ -73,4 +75,11 @@ target_link_libraries(
   velox_exec
   Boost::regex
   Folly::folly
+  ${QPL_STATIC_LINK_LIBS}
```

Comment:
should we use "qpl::qpl" here?

Reply:
Yes, please use targets unless there is a specific reason why that is not possible. And as I understand it this is an optional feature so this should be in a separate, gated call to `target_link_libraries`. (if it is even necessary when looking at L81ff in this file...)